### PR TITLE
Update operator.rs

### PR DESCRIPTION
**Pull request by januszgrze**
# Description

some spell checks

## Linked Issues

N/A

## Testing

N/A

## Docs

Not needed. just some spelling errors.

### CODE REVIEW

**Review by otaliptus — APPROVED** [review]
~LGTM~

File: core/src/operator.rs [hunk 1]
```diff
@@ -220,7 +220,7 @@ where
             // The amount will be calculated as if the transaction has 1 input
             // and (num_kickoff_utxos + 2) outputs where the first k outputs are
             // the kickoff outputs, the penultimante output is the change output,
-            // and the last output is the anyonecanpay output for fee bumping.
+            // and the last output is the anyone canpay output for fee bumping.
```
Comment (eyusufatik) [inline core/src/operator.rs:223]: I believe this is written bc there is a sighash type called ANYONECANPAY

Comment (ceyhunsen) [reply core/src/operator.rs:223]: @januszgrze can you fix this?

**Review by ceyhunsen — CHANGES_REQUESTED** [review]

**Comment by ceyhunsen** [comment]
Closing because took too long to respond.

**Pull request closed without merging**